src/components/fieldNumber/fieldNumber.tsx

```diff
@@ -49,9 +51,11 @@ export const FieldNumber: FC<FieldNumberProps> = ({
   title,
   error,
   touched = false,
+  id,
```

Comment:
Could you please explain why we need to pass an `id` here?

Reply:
Makes sense. I asked mostly to understand whether we really need to override auto-generated id for our use cases, or if it's just for flexibility.

Reply:
@AmsterGet since we can pass `HTMLInputElement` props and then spread them with `{...rest}` which includes id, the auto generated id can be overriden 
but there is no specific use cases